l10n_ch_payment_slip/i18n/de.po

```diff
@@ -57,13 +57,13 @@ msgstr "Horiz. position in mm für die Adresse"
 #: model:ir.model.fields,field_description:l10n_ch_payment_slip.field_res_company_bvr_background
 #: field:res.company,bvr_background:0
 msgid "Insert BVR background ?"
-msgstr "BVR Hintergrund einfügen ?"
+msgstr "ESR Hintergrund einfügen ?"
```

Comment:
msgstr "ESR Hintergrund einfügen?"

(normally, there is no space before a question mark / exclamation mark)